core/tests/protractor_desktop/profileFeatures.js

```diff
@@ -15,3 +15,142 @@
 /**
  * @fileoverview End-to-end tests for user profile features.
  */
+
+var DEFAULT_BIO = 'This user has not supplied a bio yet.';
+var PLACEHOLDER_INTEREST_TEXT = 'none specified';
+
+var users = require('../protractor_utils/users.js');
+var general = require('../protractor_utils/general.js');
+var waitFor = require('../protractor_utils/waitFor.js');
+
+var ProfilePage = require('../protractor_utils/ProfilePage.js');
+var PreferencesPage = require('../protractor_utils/PreferencesPage.js');
+
+describe('Un-customized profile page for current, logged-in user', function() {
```

Comment:
I think we can combine the various describe blocks here into one big describe with several it() blocks right? The setup mostly seems common across all the suites here.

Reply:
I think we should still have two `describe` blocks: one for unmodified profiles and one for modified profiles. My reasoning is that for the modified profiles, I include code to set the user's bio and interests in the `beforeAll` function, which shouldn't happen for the unmodified profiles. I do this in a564fbf.